### PR TITLE
Incremental Loading of Dimensions

### PR DESCRIPTION
**Pull request by DennisGibz**
The purpose is to replace flush and load of dimensions and fact tables with incremental loading.

The scripts aren't expected to change in logic (what is currently there).

### CODE REVIEW

**Comment by nobert-mumo** [comment]
@DennisGibz could you kindly format the scripts  to make them more readable ? the long ones loke DimPatient are hard to read.
We can use this online formatter: https://www.atatus.com/tools/sql-beautify

File: Scripts/NDWH/ALL DIMENSIONS/load_DimPatient.sql
Comment (Marymary-dev) [inline Scripts/NDWH/ALL DIMENSIONS/load_DimPatient.sql:1]: @DennisGibz  is it that you have added the other variables as matching keys?

**Comment by nobert-mumo** [comment]
> @DennisGibz could you kindly format the scripts to make them more readable ? the long ones loke DimPatient are hard to read. We can use this online formatter: https://www.atatus.com/tools/sql-beautify

@DennisGibz kindly re format the scripts to enable easier revewing.

**Comment by DennisGibz** [comment]
> @DennisGibz could you kindly format the scripts to make them more readable ? the long ones loke DimPatient are hard to read. We can use this online formatter: https://www.atatus.com/tools/sql-beautify

Well noted @nobert-mumo

**Comment by nobert-mumo** [comment]
Closing this as there is a new PR on this.

File: Scripts/NDWH/ALL DIMENSIONS/load_DimPatient.sql
Comment (DennisGibz) [reply Scripts/NDWH/ALL DIMENSIONS/load_DimPatient.sql:1]: End game is to persist the dim Keys for reusability. I am not sure if I have answered your question, right?

**Pull request closed without merging**